dockerize-nest/README.md

```diff
@@ -4,6 +4,10 @@
 - [Node](https://nodejs.org/en/download) v20.13.1 installed
   - Preferably installed via [NVM (Node Version Manager)](https://github.com/nvm-sh/nvm)
 - Git submodule [`/dockerize-nest/nest`](./nest) initiated
+- A new external docker network, use this command:
```

Comment:
Can be reverted, duplicated the `dockerized-nest` in the `basic-monitoring` folder